apps/portal/src/content/docs/components/alert-dialog.mdx

```diff
@@ -0,0 +1,341 @@
+---
+title: AlertDialog
+description: AlertDialog component documentation
+---
+
+The `AlertDialog` component provides a flexible UI element for displaying alert dialogs. It is composed of several subcomponents such as `AlertDialogHeader`, `AlertDialogFooter`, `AlertDialogTitle`, and `AlertDialogDescription` to offer a structured and customizable interface.
+
+import { DocsPage } from "@/components/docs-page";
+import { AlertDialog } from "@harnessio/ui/components";
+import { Button } from "@harnessio/ui/components";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<AlertDialog.Root>
+    <AlertDialog.Trigger asChild>
+      <Button>Open Alert Dialog</Button>
+    </AlertDialog.Trigger>
+    <AlertDialog.Content>
+      <AlertDialog.Header>
+        <AlertDialog.Title>Alert Dialog Title</AlertDialog.Title>
+      </AlertDialog.Header>
+      <AlertDialog.Description variant="secondary">
+        <p>Alert dialog content goes here. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+      </AlertDialog.Description>
+      <AlertDialog.Footer>
+        <AlertDialog.Cancel>Cancel</AlertDialog.Cancel>
+        <AlertDialog.Action>Confirm</AlertDialog.Action>
+      </AlertDialog.Footer>
+    </AlertDialog.Content>
+  </AlertDialog.Root>`}
+/>
+
+## Usage
+
+```typescript jsx
+import { AlertDialog } from '@harnessio/ui/components'
+import { Button } from '@harnessio/ui/components'
+
+return (
+  <AlertDialog.Root>
+    <AlertDialog.Trigger asChild>
+      <Button>Open Alert Dialog</Button>
+    </AlertDialog.Trigger>
+    <AlertDialog.Content>
+      <AlertDialog.Header>
+        <AlertDialog.Title>Alert Dialog Title</AlertDialog.Title>
+      </AlertDialog.Header>
+      <AlertDialog.Description variant="secondary">
+        This is the alert dialog description.
+        <p>Alert dialog content goes here.</p>
+      </AlertDialog.Description>
+      <AlertDialog.Footer>
+        <AlertDialog.Cancel>Cancel</AlertDialog.Cancel>
+        <AlertDialog.Action>Confirm</AlertDialog.Action>
+      </AlertDialog.Footer>
+    </AlertDialog.Content>
+  </AlertDialog.Root>
+)
+```
+
+## Anatomy
+
+All parts of the AlertDialog component can be imported and composed as required.
+
+```typescript jsx
+<AlertDialog.Root>
+  <AlertDialog.Trigger>
+    <Button>Open Alert Dialog</Button>
+  </AlertDialog.Trigger>
+  <AlertDialog.Content>
+    <AlertDialog.Header>
+      <AlertDialog.Title>Alert Dialog Title</AlertDialog.Title>
+    </AlertDialog.Header>
+    <AlertDialog.Description variant="secondary">
+      This is the alert dialog description.
+      <p>Alert dialog content goes here.</p>
```

Comment:
Again, only things that are allowed in a paragraph should be used in a `Description`
```suggestion
      This is the alert dialog description.
      Alert dialog content goes here.
```